src/eos-autoupdater.c

```diff
@@ -30,13 +50,17 @@ typedef enum _UpdateStep {
   UPDATE_STEP_APPLY
 } UpdateStep;
 
+/* These must be kept in sync with #UpdateStep. */
+#define UPDATE_STEP_FIRST UPDATE_STEP_NONE
+#define UPDATE_STEP_LAST UPDATE_STEP_APPLY
+
 #define SEC_PER_DAY (3600ll * 24)
 
 /* This file is touched whenever the updater starts */
-#define UPDATE_STAMP_DIR        "/var/lib/eos-updater"
-#define UPDATE_STAMP_NAME       "eos-updater-stamp"
+static const char *UPDATE_STAMP_DIR = LOCALSTATEDIR "/eos-updater";
```

Comment:
Huh, I've never heard of `localstatedir` being anything but `/var` or `$prefix/var`. Certainly autoconf never does anything besides defaulting to `$prefix/var` - http://git.savannah.gnu.org/gitweb/?p=autoconf.git;a=blob;f=lib/autoconf/general.m4;hb=HEAD#l588. Can you point me to the description of the non-GNU localstatedir definition?

I'm not sure why we'd care about a non-GNU default, either. The intention for this code (and all Endless code as far as I know) has always been to follow the GNU/FHS standards. The code had unfortunately hardcoded `/var/lib/eos-updater` before, but to me the logical fix there is to pull in the `/var` prefix rather than assuming someone would pass in a non-standard `localstatedir` with `/lib` suffixed.